internal/api/server/server.go

```diff
@@ -12,6 +13,8 @@ import (
 	"github.com/edulinq/autograder/internal/util"
 )
 
+var FinishCleanup sync.WaitGroup
```

Comment:
Comment this.
This should probably be a couple sentences to describe what it is, why we need it, and how people should use it.